src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -13,11 +13,7 @@
         <service id="cache.adapter.shared" alias="cache.adapter.filesystem" />
         <service id="cache.adapter.local" alias="cache.adapter.filesystem" />
 
-        <service id="cache.pool.shared" parent="cache.adapter.shared">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
-        <service id="cache.pool.local" parent="cache.adapter.local">
+        <service id="cache.pool.app" parent="cache.adapter.shared">
```

Comment:
At first glance, maybe this should be called `cache.pool.default`  I don't like the `.app` suffix because all the pools are for the app and because the `app.` convention is commonly used as a prefix for user's stuff.